etna/transforms/math/limit.py

```diff
@@ -0,0 +1,145 @@
+from typing import List
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+from etna.transforms.base import ReversibleTransform
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders.
```

Comment:
May be you can add in the end "... between the borders (lower_bound - toll, upper_bound + toll)"